docker/terraform/cloudbuild.yaml

```diff
@@ -0,0 +1,29 @@
+# In this directory, run the following command to build this builder.
+# $ gcloud builds submit . --config=cloudbuild.yaml
+substitutions:
+  _TERRAFORM_VERSION: 1.3.7
+  _TERRAFORM_VERSION_SHA256SUM: b8cf184dee15dfa89713fe56085313ab23db22e17284a9a27c0999c67ce3021e
+steps:
+- name: 'gcr.io/cloud-builders/wget'
+  args: ["https://releases.hashicorp.com/terraform/${_TERRAFORM_VERSION}/terraform_${_TERRAFORM_VERSION}_linux_amd64.zip"]
```

Comment:
Can we instead build from source using `go install` ? 

This reduces a bit of supply chain risk (given that these builds are not signed/verifiable), by reducing it to the GitHub repo rather than including `releases.hashicorp.com` as well and every step that led to the binary getting uploaded there.

Reply:
(Disclaimer: I didn't write this file)
Aren't these builds signed? [Hashicorp provides a signature for the checksum files they provide](https://www.hashicorp.com/security)

I can move to building with go anyway, if that's your preference.

Reply:
Missed that there are some signatures provided here for the checksums (but we're not verifying them anywhere anyway?)

That said, it's not as good as [SLSA](https://slsa.dev/) in that it doesn't verify the chain of trust all the way back to the source repo itself and reduce the risk of compromise to just the builder. The key management, security of hashicorp.com etc are all still included in our risk by relying on this current approach.

Reply:
I've rejigged this to now build with `go install`